chacklist.txt

```diff
@@ -0,0 +1,7 @@
+Arch - Rel Eval 3
```

Comment:
Did you mean to name it `checklist`?

Reply:
These are the files that are written by aws and there was a problem with calculating their score. it seems that it took a long time to load them.
yeah😁, it was a typo.

Reply:
What do you mean by "written by aws"?

Reply:
I mean they have used aws html elements. I'm not sure this is the problem though. I listed them so that we can check them later.
> <script src="https://assets.crowd.aws/crowd-html-elements.js"></script>